src/js/models/v2/awards/BaseAwardRecipient.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * BaseAwardRecipient.js
+ * Created by Kevin Li 2/22/18
+ */
+
+import { getBusinessTypes } from 'helpers/businessTypesHelper';
+import { formatMoney } from 'helpers/moneyFormatter';
+import CoreLocation from 'models/v2/CoreLocation';
+
+const parseBusinessCategories = (data) => (
+    getBusinessTypes().reduce((parsed, type) => {
+        if (data[type.fieldName]) {
+            parsed.push(type.displayName);
+        }
+        return parsed;
+    }, [])
+);
+
+const parseExecutiveCompensation = (data) => {
+    const executiveCompensation = {};
+    for (let i = 1; i < 6; i++) {
+        const name = data[`officer_${i}_name`] || '';
+        const amount = formatMoney(data[`officer_${i}_amount`]) || 0;
+        executiveCompensation[`officer${i}`] = `${name} - ${amount}`;
+    }
+    return executiveCompensation;
+};
+
+const BaseAwardRecipient = {
+    populate(data) {
+        this.internalId = data.legal_entity_id || '';
+        this.name = data.recipient_name || 'Unknown';
+        this.duns = data.recipient_unique_id || '--';
+        this.parentDuns = data.parent_recipient_unique_id || '--';
+        this._businessTypes = data.business_types_description || '';
+        this._businessCategories = parseBusinessCategories(data);
+
+        // Recipient Location
+        let location = {};
```

Comment:
Should `location` default to `{}` or would it be better to use `null` so we can use truthy checks to see if it's available?
Alternatively, should it just populate a blank CoreLocation object if there's no location data provided?